build_iso/sles12.rb

```diff
@@ -65,12 +72,13 @@
   system "find . -name \"*.rpm\"|xargs mkdud -c #{version}.dud -d sle12 -i instsys,repo --prefix=37"
 
   puts "\n**** Creating new ISO image with the updated packages ****"
-  system "sudo mksusecd -c testing.iso --initrd=#{version}.dud #{iso_path}"
+puts "sudo mksusecd -c testing.iso --initrd=#{version}.dud #{iso_path} #{boot_dir}"
+  system "sudo mksusecd -c testing.iso --initrd=#{version}.dud #{iso_path} #{boot_dir}"
```

Comment:
I'd just use a variable to store the whole command and then I will call `puts` and `system` just passing that variable (so we won't forget to update both lines when something changes).